helpers/middlewares/limit_context.js

```diff
@@ -0,0 +1,26 @@
+let MAX_CONTEXT_COUNTER;
+let contextCounter = 0;
+
+function incContextCounter() {}
+exports.incContextCounter = incContextCounter;  // Empty function or incrementer
+
+function decContextCounter() {}
+exports.decContextCounter = decContextCounter;  // Empty function or decrementer
+
+exports.initContextCounter = function (maxContextCounter) {
+    if (!isNaN(maxContextCounter)) {
+        MAX_CONTEXT_COUNTER = maxContextCounter;
+        exports.incContextCounter = () => { contextCounter++ };
+        exports.decContextCounter = () => { contextCounter-- };
+    }
+}
+
+exports.limitContextMiddleware = async function limitContextMiddleware(req, res, next) {
+    if (contextCounter > MAX_CONTEXT_COUNTER) {
```

Comment:
With this approach, request creating a context that exceeds the limit is allowed, but next requests are failed. Consider the case when you create limit + 1 contexts, and then send following goto request to one of them - it will fail with 429. But requests to existing contexts should not be affected by the limit.